### DISCUSSION TITLE
elastalert python WARNING:elasticsearch:DELETE https://[es_host]/_search/scroll [status:404 request:0.013s]

### DISCUSSION THREAD

**Discussion by Ox0000ff** [jertel/elastalert2#1347] [Q&A] [answered]
I have an elastalert2 instance querying an elastic cloud stack, once I try to test alerts with a simple rule, I'm getting 0 hits contiously followed by this warning:

**WARNING:elasticsearch:DELETE https://[es_host]/_search/scroll [status:404 request:0.016s]**

Even having matching events (checked in UI) I'm getting 0 hits and no alert is generated.

The same setup is working properly from a different machine, both of them using elastalert 0.2.15 but never running at the same time.

What I`ve tried:
- Check all comms
- Reinstalled elastalert2 and dependencies just to ensure
- Modified max_query_value and scroll_keepalive values
- Reduce parallel execution

Do you have any clue on this? I've seen the same problem in other post but related with two instances running at the same time
Thanks in advance

**Comment by jertel** [comment]
To clarify:

- Do you have two separate problems here? 1) DELETE warnings, and 2) no hits?
- Are you claiming that one ElastAlert2 instance works fine against Server "A", but another ElastAlert2 instance experiences both problems mentioned in my above bullet when also using Server "A"?

**Reply by Ox0000ff** [reply]
Thanks for your answer:
- I'm having those two problems with the same instance and I'm guessing the two problems are related.
- To your second question, exactly like you said. The second instance is having these two problems, we've checked communications again. First one is working as expected

I think there is something related to the second machine we've built, but honestly I'm running out of ideas for checking it as we are not having communication issues and seems not to be about elastic configuration.

**Reply by jertel** [reply, marked as answer]
It seems like the two ElastAlert instances are either accessing different Elasticsearch clusters, or they have different access rights. For example, the ElastAlert instance that isn't working perhaps doesn't have read/write/scroll/etc rights to the cluster, but the working ElastAlert instance does have those privileges.

I can't think of any other reason why identical queries originating from alternate sources would see different results.

If I were in your shoes I would use tcpdump or wireshark to trace the packets and confirm that they are arriving from both ElastAlert instances.
